src/components/htmlvideoplayer/plugin.js

```diff
@@ -1071,6 +1101,14 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
             });
         }
 
+        function renderSsaAss(videoElement, track, item) {
+            if (supportsCanvas() && supportsWebWorkers()) {
+                renderWithSubtitlesOctopus(videoElement, track, item);
+            } else {
+                renderWithLibjass(videoElement, track, item);
```

Comment:
I was told that libjass wasn't working right now, is that not the case?

Reply:
libjass works for me, but the subtitles are rendered as though the window height is the video height, so they can end up huge and behind the OSD if the window is narrow.

Reply:
So from now on libjass will be used as a fallback for ssa and ass subtitles?